### PR TITLE
Rename all the import aliases to make them consistent across the project

### LINKED ISSUE #2938: internal: Update Contour import alias for Contour API
The Contour import alias for the contour api is currently `contourv1`. Since the refactoring the internal/contour package into `contourv2` and it's not clear which version we should use.  

We could use `contourv1api` or similar for the the Contour api. 

_Originally posted by @skriss in https://github.com/projectcontour/contour/pull/2928#discussion_r493837575_

### PR DESCRIPTION
**Pull request by stevesloka**
Renames all the import aliases to make them consistent across the project.
    
Fixes #2938
    
Signed-off-by: Steve Sloka <slokas@vmware.com>

### CODE REVIEW

**Comment by stevesloka** [comment]
Blocked waiting on #2949 to merge first.

File: examples/contour/02-role-contour.yaml [hunk 1]
```diff
@@ -48,61 +48,61 @@ rules:
...
-  - gateways
-  - httproutes
-  - tcproutes
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups:
-  - networking.k8s.io
+  - contour_api_v1.io
```
Comment (skriss) [inline examples/contour/02-role-contour.yaml:51]: looks like this file needs to be reverted

File: examples/render/contour.yaml [hunk 2]
```diff
@@ -1540,61 +1540,61 @@ rules:
...
-  - gateways
-  - httproutes
-  - tcproutes
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups:
-  - networking.k8s.io
+  - contour_api_v1.io
```
Comment (skriss) [inline examples/render/contour.yaml:1543]: revert this file

File: internal/k8s/informers.go [hunk 3]
```diff
@@ -26,17 +25,17 @@ import (
 // +kubebuilder:rbac:groups="networking.k8s.io",resources=ingresses,verbs=get;list;watch
 // +kubebuilder:rbac:groups="networking.k8s.io",resources=ingresses/status,verbs=create;get;update
 
-// +kubebuilder:rbac:groups="projectcontour.io",resources=httpproxies;tlscertificatedelegations,verbs=get;list;watch
-// +kubebuilder:rbac:groups="projectcontour.io",resources=httpproxies/status,verbs=create;get;update
-// +kubebuilder:rbac:groups="projectcontour.io",resources=extensionservices,verbs=get;list;watch
-// +kubebuilder:rbac:groups="projectcontour.io",resources=extensionservices/status,verbs=create;get;update
+// +kubebuilder:rbac:groups="contour_api_v1.io",resources=httpproxies;tlscertificatedelegations,verbs=get;list;watch
```
Comment (skriss) [inline internal/k8s/informers.go:28]: need to undo these changes

**Review by skriss — CHANGES_REQUESTED** [review]

File: examples/contour/02-role-contour.yaml (hunk 1 above)
Comment (stevesloka) [reply examples/contour/02-role-contour.yaml:51]: 👍

**Review by skriss — APPROVED** [review]
LGTM
